scripts/tooling.py

```diff
@@ -32,8 +32,7 @@ def lint():
 
 
 def test():
-    _run("python -u -m unittest discover tests")
-    _run("python -u -m unittest discover parsers/test")
+    _run("pytest")
```

Comment:
Ideally we would have added pytest in a separate PR.

Does that still cover all of the other tests?

Reply:
> Ideally we would have added pytest in a separate PR.

Yea, I know 🙈

>Does that still cover all of the other tests?

Yup!